CHANGELOG.md

```diff
@@ -102,6 +102,7 @@ ___
 - EXPERIMENTAL: Added custom routes to easily customize flows for password reset, email verification or build entirely new flows (Manuel Trezza) [#7231](https://github.com/parse-community/parse-server/issues/7231)
 - Added Deprecation Policy to govern the introduction of braking changes in a phased pattern that is more predictable for developers (Manuel Trezza) [#7199](https://github.com/parse-community/parse-server/pull/7199)
 ### Other Changes
+- Support aggregation stage names starting with '$' just like in MongoDB (Raschid JF Rafeally) [#7339](https://github.com/parse-community/parse-server/pull/7339)
```

Comment:
Depending on how you want to proceed, this either goes under the breaking changes section (without deprecation) or notable change (with deprecation).

Reply:
I'll go for the deprecation warning. We all appreciate lowering the breaking change amount.